liquibase-core/src/main/java/liquibase/command/core/UpdateCountCommandStep.java

```diff
@@ -48,11 +53,30 @@ public class UpdateCountCommandStep extends AbstractCliWrapperCommandStep {
         CONTEXTS_ARG = builder.argument("contexts", String.class)
                 .description("Changeset contexts to match").build();
         COUNT_ARG = builder.argument("count", Integer.class).required()
-            .description("The number of changes in the changelog to deploy").build();
+                .description("The number of changes in the changelog to deploy").build();
         CHANGE_EXEC_LISTENER_CLASS_ARG = builder.argument("changeExecListenerClass", String.class)
-            .description("Fully-qualified class which specifies a ChangeExecListener").build();
+                .description("Fully-qualified class which specifies a ChangeExecListener").build();
         CHANGE_EXEC_LISTENER_PROPERTIES_FILE_ARG = builder.argument("changeExecListenerPropertiesFile", String.class)
-            .description("Path to a properties file for the ChangeExecListenerClass").build();
+                .description("Path to a properties file for the ChangeExecListenerClass").build();
+        SHOW_SUMMARY = builder.argument("showSummary", UpdateSummaryEnum.class)
+                .description("Type of update summary to show.  Values can be 'off', 'summary', or 'verbose'.")
```

Comment:
I think this description would benefit from adding the word "results." "Type of update results summary to show."

Reply:
Done